### PR TITLE
Add/update some more papers

### PR DESCRIPTION
**Pull request by agrayver**

### CODE REVIEW

File: publications-2021.bib [hunk 1]
```diff
@@ -36,5 +36,25 @@ @Article{dealii2019design
...
+  year={2021},
+  publisher={Elsevier}
+}
+
+@article{grayver2021td,
+author = {Grayver, Alexander V. and Kuvshinov, Alexey and Werthm{\"u}ller, Dieter},
+title = {Time-domain modeling of 3-D Earth's and planetary electromagnetic induction effect in ground and satellite observations},
+journal = {Journal of Geophysical Research: Space Physics},
+volume = {129},
+pages = {e2020JA028672},
```
Comment (drwells) [inline publications-2021.bib:54]: Are these pages correct? I don't understand why this is not a pair of numbers.

Comment (agrayver) [reply publications-2021.bib:54]: It is in the "published online" stage, where the pages are not known yet, but the publisher gives this thing... (I exported ciation from their website...)

Comment (drwells) [reply publications-2021.bib:54]: Weird. Well, if the CI doesn't complain lets go for it.

**Review by drwells — APPROVED** [review]

**Pull request merged**